helm/cloud-provider-vsphere/values.yaml

```diff
@@ -37,3 +41,19 @@ kube-vip-cloud-provider:
   image:
     repository: docker.io/giantswarm/kube-vip-cloud-provider
     tag: "v0.0.4"
+  nameOverride: kube-vip-cloud-provider
+
+vsphere-csi-driver:
```

Comment:
I wouldn't have this at the root level. I would have it at the subchart level to keep root values clean.